qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -26,6 +26,103 @@
 
 import pennylane as qml
 from pennylane import Hamiltonian
+from pennylane.wires import Wires
+
+
+def _proc_wires(wires, n_wires=None):
+    r"""
+    Checks and processes custom user wire mapping into a consistent, direction-free, Wires format.
+    Used for converting between OpenFermion qubit numbering and Pennylane wire labels.
+
+    Since OpenFermion's quibit numbering is always consecutive int, simple iterable types such as
+    list, tuple, or Wires can be used to specify the qubit<->wire mapping with indices acting as
+    qubits. Dict can also be used as a mapping, but does not provide any advantage over lists other
+    than the ability to do partial mapping/permutation in the qubit->wire direction.
+
+    It is recommended pass Wires/list/tuple `wires` since it's direction-free, i.e. the same `wires`
+    argument can be used to convert both ways between OpenFermion and Pennylane. Only use dict for
+    partial or unordered mapping.
+
+    **Example usage:**
+
+    >>> # consec int wires if no wires mapping provided, ie. identity map: 0<->0, 1<->1, 2<->2
+    >>> _proc_wires(None, 3)
+    <Wires = [0, 1, 2]>
+
+    >>> # List as mapping, qubit indices with wire label values: 0<->w0, 1<->w1, 2<->w2
```

Comment:
I was struggling with this too 😆. I ended up having ints on the left is because, for qchem, while the conversion can happen both ways, the most frequent direction is from OpenFermion observable to Pennylane ansatz. So most of the times, we are doing the int -> wire label conversion. And after the conversion, all ints will be gone and we are left with a observable properly wired to the ansatz.